src/Kdump.ycp

```diff
@@ -981,15 +1006,12 @@ boolean WriteKdumpBootParameter () {
 	    }
 	}
         // write crashkernel option to boot section
-	if (section_pos == -1)
+	if (!is_xen()) // boot section is NOT Xen
 	{
             result = Bootloader::setKernelParam (actual_boot_section, "crashkernel", crash_value);
-	} else {
-	    BootCommon::sections[section_pos, "xen_append"]= "crashkernel="+convertCrashkernelForXEN(crash_value);
-	    // added flag which means that section was changed bnc #432651
-	    BootCommon::sections[section_pos, "__changed"] = true;
-	    result = true;
-	    y2milestone("Write boot section to XEN boot section %1", BootCommon::sections[section_pos]:nil);
+	} else { // boot section is Xen
+        UpdateXenAppendParameter("crashkernel", convertCrashkernelForXEN(crash_value));
+        result = true
```

Comment:
NP: `result` is set by default to `true`, see above. It seems you could even remove this line...